dev_guide/builds.adoc

```diff
@@ -1184,6 +1190,80 @@ spec:
 ----
 ====
 
+[[build-hooks]]
+
+== Build Hooks
+Build hooks can be used as a part of the build flow to run tests inside a temporary container and validate your image before it is being pushed to the registry. The command is executed with the current working directory set to the image's `WORKDIR`.
```

Comment:
- [ ] "before it is pushed to a registry."
- [ ] I'm not 100% sure, but I think it might be \`\*WORKDIR\*\`, here and everywhere else (AsciiDoc syntax).
- [ ] You can also add that "WORKDIR" is normally where your code resides (with your own words), as that will make it less puzzling what is "WORKDIR" and what does that actually mean.
- [ ] I suggest breaking these long lines into multiple lines, as that's the pattern we follow in `.adoc` files. See the paragraphs above and below the new content. With Atom, ctrl-alt-q on a line will reformat the current paragraph.